main.js

```diff
@@ -1,73 +1,205 @@
 /* eslint-disable linebreak-style */
 import Book from './module/constructor.js';
 import Store from './module/store.js';
-// UI Class: Handle UI Tasks
-class UI {
-  static displayBooks() {
-    const books = Store.getBooks();
+import Navigation from './module/navigation.js';
 
-    books.forEach((book) => UI.addBookToList(book));
-  }
+const date = window.luxon;
 
-  static addBookToList(book) {
-    const list = document.querySelector('#book-list');
-    const row = document.createElement('tr');
-    // element.classList.add('table-row');
-    row.innerHTML = `
-      <td class="book-title">"${book.title}" </td>
-      <td> by</td>
-      <td>${book.author}</td>
-      <td>
-      <input class='btn btn-danger btn-sm delete' type="button" value="Remove">
-     </td>
-    `;
-    list.appendChild(row);
+class UI {
+  static displayBook() {
+    const bookList = Store.getBooks();
+    bookList.forEach((book) => UI.addBookToList(book));
   }
 
-  // delete function
-  static deleteBook(el) {
-    if (el.classList.contains('delete')) {
-      el.parentElement.parentElement.remove();
-    }
+  static addBookToList(book) {
+    const listOutput = document.querySelector('.listDisplay');
+    listOutput.innerHTML += `<div class="book rounded">
+                               <div class="book-infos py-2" id="${book.id}">   
+                                 <span class="px-2 fw-bolder item1 blck text-capitalize">"${book.title}"</span>
+                                 <span id="by" class="px-2">by</span>
+                                 <span id="spanAuthor" class="px-2 item3 blck text-capitalize"><span id="by1">by:</span> ${book.author}</span>
+                                 <button id="removeBtn" class="position-absolute h5 py-1 px-4 removeBtn clickable rounded end-0 bottom-0" >Delete</button>
+                               </div>
+                             </div>
+                            `;
   }
 
-  // Create clear fields function
-  static clearFields() {
-    document.querySelector('#title').value = '';
-    document.querySelector('#author').value = '';
+  static deleteBook(id) {
+    const el = document.getElementById(`${id}`);
+    el.parentNode.removeChild(el);
   }
 }
 
-// Event: Display Books
-document.addEventListener('DOMContentLoaded', UI.displayBooks);
+function addedSuccess() {
+  document.getElementById('addMsg').style = 'display:block';
+  document.getElementById('addMsg').innerHTML = 'Book added successfully!!!';
+  setTimeout(() => {
+    document.getElementById('addMsg').style = 'display:none';
+  }, 2000);
+}
 
-// Event: Add a Book
-document.querySelector('#book-form').addEventListener('submit', (e) => {
-  // Prevent actual submit
+document.addEventListener('submit', (e) => {
   e.preventDefault();
+  // if (e.target.title.value === '' || e.target.author.value === '') {
+  //   alert('Input field cannot be empty');
+  // // } else {
```

Comment:
- [ ] Kindly, remember not to leave commented code 🍡, always you should maintain best practices in JavaScript. Please, remove commented code 🔢.

Reply:
@fherrerao 
Removed commented code at this [commit](https://github.com/Lucash2022/Awesome_Book/commit/7d237603c0d4e20b0888fbbd42b53f3034bfca1a)